idv-front/src/stores/documentStore.js

```diff
@@ -126,37 +78,53 @@ export class DocumentStore {
         if (!getItem(KEYS.SESSION_ID)) {
           const countryCode = getItem(KEYS.COUNTRY_CODE);
           const docType = getItem(KEYS.DOC_TYPE);
+
+          const {
+            data: { data: gipsIdentity },
+          } = await API.post(APIRoutes.createLivenessIdentity);
+
+          const consentsPayload = [
+            {
+              approved: true,
+              type: 'PORTRAIT',
+            },
+          ];
+
+          // if SOR check check consent GIV
+          if (JSON.parse(getItem(KEYS.SOR_CHECK))) {
```

Comment:
This is weird condition - I suppose that `getItem(KEYS.SOR_CHECK)` will be a boolean'ish value so this will return boolean? If this is not boolean, JSON.parse from '' will throw an exception.